### PR TITLE
Add Download Link & Contact for Indian Embassy.

### PR DESCRIPTION
**Pull request by ikennaokpala**
PT Story:  https://www.pivotaltracker.com/projects/1270592/stories/115617999

* Change Contact details for Britain Embassy in India

### Expected changes

*  [URL on Gov.UK](//gov.uk/marriage-abroad/y/india/uk/partner_local/opposite_sex)
* * Change Contact details for Britain Embassy in India

### Before

screen shot 2016-03-14 at 17 49 09

### After

screen shot 2016-03-14 at 17 52 00

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb [hunk 1]
```diff
@@ -43,11 +43,36 @@
     <% end %>
   <% when 'india' %>
 
-    The British High Commission in India can provide you with a letter explaining this position for a charge - see [consular fees for India](/government/publications/india-consular-fees).
+    Download and print the [information note on ‘no objection certificates’](https://www.gov.uk/government/publications/marriage-information-note-on-noc) and show it to the registrar.
```
Comment (erikse) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb]: I think govspeak will support relative URLs for ones like in this content?

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb [hunk 2]
```diff
@@ -43,11 +43,36 @@
...
+
+    Defence fax: +91 (11) 2419 2252
+
+    Political fax: +91 (11) 2419 2464
+
+    Consular fax: +91 (11) 2419 2491 (not for visa issues)
+
+    Email: [web.newdelhi@fco.gov.uk](mailto:web.newdelhi@fco.gov.uk)
+
+    [British High Commission New Delhi - opening hours](https://www.gov.uk/government/world/organisations/british-high-commission-new-delhi/office/british-high-commission-new-delhi)
```
Comment (erikse) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb]: Relative URL: maybe in the line above also?

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb [hunk 3]
```diff
@@ -43,11 +43,36 @@
...
+
+    Economic & Commercial fax: +91 (11) 2419 2301
+
+    Defence fax: +91 (11) 2419 2252
+
+    Political fax: +91 (11) 2419 2464
+
+    Consular fax: +91 (11) 2419 2491 (not for visa issues)
+
+    Email: [web.newdelhi@fco.gov.uk](mailto:web.newdelhi@fco.gov.uk)
```
Comment (erikse) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb]: I think this line could be written (removing the duplication of writing the address twice) as:
```
Email: <mailto:web.newdelhi@fco.gov.uk>
```

**Comment by erikse** [comment]
In the ZenDesk ticket, I don't see any mention of changing the address - it would be good to clarify where this requirement came from, either by adding a reference to source of that, or adding it to the description.

Other than that, and the minor comments on the markdown, this looks fine

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb (hunk 1 above)
Comment (ikennaokpala) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb]: Change made in this [commit](https://github.com/alphagov/smart-answers/pull/2365/commits/5037c0f38a4fc1cd76f25849e99e5938d3209c1a)

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb (hunk 3 above)
Comment (ikennaokpala) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb]: @erikse  Change made in this [commit](https://github.com/alphagov/smart-answers/pull/2365/commits/78f0e00b43cc0244a0fa6ecb3ee86348df793f5b)

**Comment by ikennaokpala** [comment]
@erikse @pmanrubia factcheck for this has comeback

**Comment by ikennaokpala** [comment]
@erikse si stephens requested that this changes be made..

**Comment by erikse** [comment]
This looks good to merge

**Pull request merged**